src/test/java/uk/gov/justice/hmpps/prison/dsl/NomisData.kt

```diff
@@ -16,14 +16,7 @@ class NomisDataBuilder(
 ) {
   fun build(dsl: NomisData.() -> Unit) {
     SecurityContextHolder.setContext(
-      SecurityContextHolder.createEmptyContext().apply {
-        val principal = User("ITAG_USER", "", true, true, true, true, emptyList())
-        this.authentication = UsernamePasswordAuthenticationToken.authenticated(
```

Comment:
don't create username password authentication token, instead manually call the mock user annotation.  Tried adding the annotation to the method but didn't do anything